lib/propolis/src/hw/nvme/mod.rs

```diff
@@ -411,6 +411,21 @@ impl NvmeCtrl {
         b << (self.ns_ident.lbaf[(self.ns_ident.flbas & 0xF) as usize]).lbads
     }
 
+    /// Check a given request transfer size against any MDTS configured on the
+    /// controller.
+    ///
+    /// Returns `true` if the size is acceptable for the MDTS.
+    fn valid_for_mdts(&self, sz: u64) -> bool {
+        match self.ctrl_ident.mdts {
+            0 => true,
+            mdts => {
+                // MPSMIN is currently locked to PAGE_SIZE
```

Comment:
nit: I know we assume `PAGE_SIZE` various places but can just grab `MPSMIN` off of `self` here so there's one less place to remember.

Reply:
Done.